modules/getting-started/pages/do-a-quick-install.adoc

```diff
@@ -25,7 +22,7 @@ Open a console window on your computer and enter the following command:
 
 [source,console]
 ----
-docker run -t --name db -p 8091-8096:8091-8096 -p 11210-11211:11210-11211 couchbase/server:enterprise-7.2.0
+docker run -t --name db -p 8091-8096:8091-8096 -p 11210-11211:11210-11211 couchbase/server:enterprise
```

Comment:
Checked and it works. Suggest adding a line break because it looks bad having the command overflow the text box.

```suggestion
docker run -t --name db -p 8091-8096:8091-8096 -p 11210-11211:11210-11211 \
  couchbase/server:enterprise
```